cluster-up/cluster/kind/common.sh

```diff
@@ -52,11 +52,13 @@ function _wait_containers_ready {
 }
 
 function _fetch_kind() {
-    if [ ! -f ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kind ]; then
-        curl -LSs  https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-linux-${ARCH} -o ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kind
-        chmod +x ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kind
+    KIND="${KUBEVIRTCI_CONFIG_PATH}"/"$KUBEVIRT_PROVIDER"/.kind
+    current_kind_version=$($KIND --version |& awk '{print $3}')
```

Comment:
On my kind, I see this:

```
[~] /tmp/gp/bin/kind version
kind v0.10.0 go1.15.2 linux/amd64
[~] /tmp/gp/bin/kind version |& awk '{ print $3 }'
go1.15.2
```

Is you output different? With this, I assume we would always use $KIND_VERSION since @current_kind_version would never match

Reply:
Its hardcoded to check only the one at _ci-configs ($KUBEVIRTCI_CONFIG_PATH)
not any other one.
because of this line `KIND="${KUBEVIRTCI_CONFIG_PATH}"/"$KUBEVIRT_PROVIDER"/.kind`
This one is totally managed by kubevirtci,
so the mechanism would know to download it if it doesn't exists, and also move back and forth versions in case those are changed by overriding the parameters.
(the download is pretty fast)

Reply:
Gotcha, missed --